cypress-tests/cypress.json

```diff
@@ -0,0 +1,16 @@
+{
+    "defaultCommandTimeout": 10000,
+    "viewportWidth": 1920,
+    "viewportHeight": 1080,
+
+    "env": {
+        "test_user_name": "admin",
+        "test_user_password": "admin",
+        "api_endpoint": "http://che-eclipse-che.172.19.20.205.nip.io/api/",
```

Comment:
hello, I'm not sure these addresses can be hardcoded ?

Reply:
Thank you for your attention. On this iteration, it is OK for us. For CI we will pass this variable value by CLI command, and for a local run with dashboard it requires just editing of this variable and, "root_url".

Reply:
Small update. This variables were reworked. For now, it requires just one  "baseUrl" variable.